docs/supported_inventory_types.md

```diff
@@ -1,97 +1,156 @@
+| PHP        | Composer                                  | `php/composerlock`                   |
+| Python     | Installed PyPI packages (global and venv) | `python/wheelegg`                    |
+|            | requirements.txt                          | `python/requirements`                |
+|            | poetry.lock                               | `python/poetrylock`                  |
+|            | Pipfile.lock                              | `python/pipfilelock`                 |
+|            | pdm.lock                                  | `python/pdmlock`                     |
+|            | Conda packages                            | `python/condameta`                   |
+|            | setup.py                                  | `python/setup`                       |
+| R          | renv.lock                                 | `r/renvlock`                         |
+| Ruby       | Installed Gem packages                    | `ruby/gemspec`                       |
+|            | Gemfile.lock (OSV)                        | `ruby/gemfilelock`                   |
+| Rust       | Cargo.lock                                | `rust/cargolock`                     |
+|            | Cargo.toml                                | `rust/cargotoml`                     |
+|            | Rust binaries                             | `rust/cargoauditable`                |
+| Swift      | Podfile.lock                              | `swift/podfilelock`                  |
+|            | Package.resolved                          | `swift/packageresolved`              |
 
 ## Misc
 
-* Wordpress plugins
-  * Installed plugins
-* VSCode extensions
-* Chrome extensions
+| Type              | Extractor Plugin    |
+|-------------------|---------------------|
+| Wordpress plugins | `wordpress/plugins` |
+| VSCode extensions | `vscode/extensions` |
+| Chrome extensions | `chrome/extensions` |
+
+### Container inventory
+
+| Type                        | Extractor Plugin                                                                   |
+|-----------------------------|------------------------------------------------------------------------------------|
+| Containerd container images | `containers/containerd-runtime` (standalone), `containers/containerd` (filesystem) |
+| Docker container images     | `containers/docker` (standalone)                                                   |
+| Podman container images     | `containers/podman` (filesystem)                                                   |
+
+### SBOM files
+
+| Type                       | Extractor Plugin |
+|----------------------------|------------------|
+| SPDX SBOM descriptors      | `sbom/spdx`      |
+| CycloneDX SBOM descriptors | `sbom/cdx`       |
+
+If you're a SCALIBR user and are interested in having it support new inventory types we're happy to accept
+contributions. See the docs on [how to add a new Extractor](/docs/new_extractor.md).
+
+## Enrichers
```

Comment:
How about we order the plugins as Extractors, Detectors, Annotators, Enrichers? Then they're in decreasing order by number of plugins for the given type